File: movies_repository/src/main/java/com/nikita/popularmovies/common/models/Section.java
package com.nikita.popularmovies.common.models;

import android.support.annotation.StringDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import static android.provider.Telephony.TextBasedSmsColumns.PERSON;
import static com.nikita.popularmovies.common.models.Section.MOVIE;
import static com.nikita.popularmovies.common.models.Section.TV_SHOW;


@Retention(RetentionPolicy.SOURCE)
@StringDef(value = { MOVIE, TV_SHOW, PERSON })
public @interface Section {
  String MOVIE = "movie";
  String TV_SHOW = "tv";
  String PERSON = "person";
}

File: movies_repository/src/main/kotlin/com/nikita/popularmovies/common/models/Section.kt
package com.nikita.popularmovies.common.models

import android.support.annotation.StringDef

@Retention(AnnotationRetention.SOURCE)
@StringDef(MOVIE, TV_SHOW, PERSON)
annotation class Section

const val MOVIE = "movie"
const val TV_SHOW = "tv"
const val PERSON = "person"